extensions/ui/org.eclipse.smarthome.ui.paper/bower.json

```diff
@@ -26,7 +26,10 @@
     "bootstrap": "~3.3.2",
     "tinycolor": "~1.1.2",
     "masonry": "~3.3.1",
-    "roboto-fontface": "~0.4.3"
+    "roboto-fontface": "~0.4.3",
+    "jquery-ui": "~1.11.0",
```

Comment:
Latest version is 1.11.4, why are you chosing an older one? Would version 1.11.2 be ok for you? This is already approved for another Eclipse project, so it would be very easy for us to us it as well.